build

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
If the style guide says so, but this script would fail on, for example, FreeBSD, where bash is found at /usr/local/bin/bash.  I suspect the intent of the the line "Executables must start with #!/bin/bash" is less about absolutely using that exact string of characters than in being specific about the shell the script should be using.  Anyway....